client/packages/portal-core/src/context-selector/utils.ts

```diff
@@ -0,0 +1,8 @@
+export function getContextPageUrl(contextId?: string): string {
+  if (!contextId) return "context-page/"
+
+  const params = new URLSearchParams()
+  params.append("contextId", contextId)
+  
+  return `context-page/$${params.toString()}`
```

Comment:
This would remove other cgi parameters

Reply:
yeah it would its okey for now but we may need to change in the future. But i dont see why its a problem now